guides/dev/package_managers/README.md

```diff
@@ -28,6 +28,41 @@ It is thus recommended to use of the following strategies:
 
 Both options are explained in more detail in the [Setting CKEditor Configuration](http://docs.ckeditor.com/#!/guide/dev_configuration) article.
 
+## npm
+
+This article assumes that you have **npm** already installed and added to your `PATH`. If this is not the case, please follow the [installation guide](https://docs.npmjs.com/getting-started/installing-node).
```

Comment:
We could make more meaningful link innertext here - "npm installation guide" will be much better.